docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -13,29 +13,33 @@ The main features of the Media Controller API include:
 
 - Setting up the client and server pair
 
-  You can [set up the client and server pair](#getting-the-client-and-server) by creating a new server using the `createServer()` method. On the client side, you can get a client and find active servers using the `getClient()` and `findServers()` methods.
+  You can [set up the client and server pair](#getting-client-and-server) by creating a new server using the `createServer()` method. On the client side, you can get a client and find active servers using the `getClient()` and `findServers()` methods.
```

Comment:
Please add "the" before "findServers() methods".

For Line no. 413 and 106:

Replace period (.) with a colon (:). This is to maintain consistency. Use a colon, if a code snippet is followed by a sentence.

This is not under the changed portion. If this change is under your scope. Please do fix this. If not then ignore my comments.

The other updated changes look fine.
Thank you.

Reply:
Those issues are outside the scope of this review.